### ISSUE TITLE
Any plans for more releases?

### ISSUE THREAD

**Issue by negomi** [webpack-contrib/svg-inline-loader#61]
It seems like PRs are still being merged into master, but the most recent release to npm was back in September 2016.

I'm running into this issue: https://github.com/webpack-contrib/svg-inline-loader/issues/41

**Comment by evilebottnawi** [comment]
@negomi What kind of changes were made but not published? https://github.com/webpack-contrib/svg-inline-loader/pull/42 ?

**Comment by negomi** [comment]
Yep, #42 contains the fix I need, but the commit history shows other changes too: https://github.com/webpack-contrib/svg-inline-loader/compare/779f77ec915f1218cce636cfa8d62ed58abbf317...162af5b75d9faf82c539430d9147dea9cf0c07cd

**Comment by evilebottnawi** [comment]
@negomi I will pass this request contributors of this loader. Thanks for helping us!

**Comment by negomi** [comment]
Thanks! :)

**Comment by mitkaaa** [comment]
@evilebottnawi Hi! please look to [#57](https://github.com/webpack-contrib/svg-inline-loader/pull/57)

**Comment by d3viant0ne** [comment]
@negomi - Sorry, we have been a bit busy with side effects of the Webpack v3 release.

So to answer your question, yes. Currently waiting on me to finish #55 which is next on my list now that extract-text has stabilized. I'm aiming to have it done this week.

**Comment by d3viant0ne** [comment]
Alright, change of plans. I'm going to get the fixes in master out in this major version range prior to adding the breaking changes coming with webpack-defaults.

e.t.a ~90 minutes.

**Comment by d3viant0ne** [comment]
`svg-inline-loader@0.8.0` is now out & available on npm.

Changelog for 0.8.0 can be found here - https://github.com/webpack-contrib/svg-inline-loader/blob/master/CHANGELOG.md#080-2017-07-16
